import-service/handlers/import-file-parser/parse-s3-csv-obj.ts

```diff
@@ -0,0 +1,19 @@
+import * as AWS from "aws-sdk";
```

Comment:
I suggest to move this file into `utils`.

Reply:
done